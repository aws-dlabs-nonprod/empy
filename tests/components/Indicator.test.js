import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import * as PersonaState from '@constants/PersonaState';
import { Indicator } from '@components/Indicator';

describe('Indicator', () => {

    test('renders correctly', () => {
        const component = shallow(
            <Indicator />
        );

        expect(component).toMatchSnapshot();
    });
    

    test('renders speaking', () => {
        const component = shallow(
            <Indicator personaState={ PersonaState.STATE_SPEAKING } />
        );

        expect(component).toMatchSnapshot();
    });

});
