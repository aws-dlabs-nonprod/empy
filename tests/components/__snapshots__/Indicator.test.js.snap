// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indicator renders correctly 1`] = `
<Indicator__StyledIndicator
  isAnimating={false}
>
  <span />
  <span />
  <span />
  <span />
  <span />
  <span />
</Indicator__StyledIndicator>
`;

exports[`Indicator renders speaking 1`] = `
<Indicator__StyledIndicator
  isAnimating={true}
>
  <span />
  <span />
  <span />
  <span />
  <span />
  <span />
</Indicator__StyledIndicator>
`;
